apps/app/components/ui/multi-level-dropdown.tsx

```diff
@@ -1,10 +1,18 @@
 import { Fragment, useState } from "react";
```

Comment:
Don't change the global dropdown component, if required, create a new component.